### PR TITLE
feat: add prisma schematic

### PR DESCRIPTION
**Pull request by benjamin1555**

### CODE REVIEW

File: libs/schematics/src/generators/prisma-library/helpers/project-targets.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            command: 'npx prisma generate',
+            forwardAllArgs: false,
+          },
+        ],
+        cwd: options.projectRoot,
+        parallel: false,
+      },
+    },
+    pregenerate: {
+      executor: './dist/libs/schematics:generate',
```
Comment (floross) [inline libs/schematics/src/generators/prisma-library/helpers/project-targets.ts]: Use `@tractr/schematics:generate` instead

File: libs/schematics/src/generators/prisma-library/helpers/project-targets.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    },
+    seed: {
+      executor: '@nx-tools/nx-prisma:seed',
+      options: {
+        script: `${options.projectRoot}/prisma/seed.ts`,
+        tsConfig: `${options.projectRoot}/tsconfig.json`,
+      },
+    },
+    status: {
+      executor: '@nx-tools/nx-prisma:seed',
```
Comment (floross) [inline libs/schematics/src/generators/prisma-library/helpers/project-targets.ts]: `seed`?

The documentation of nx-prisma doesn't use the right target it should be status

File: libs/schematics/src/generators/prisma-library/helpers/project-targets.ts [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      },
+    },
+    pull: {
+      executor: '@nx-tools/nx-prisma:rollback',
+      options: {
+        schema: `${options.projectRoot}/prisma/schema.prisma`,
+      },
+    },
+    push: {
+      executor: '@nx-tools/nx-prisma:seed',
```
Comment (floross) [inline libs/schematics/src/generators/prisma-library/helpers/project-targets.ts]: should be `push`

File: libs/schematics/src/generators/prisma-library/helpers/project-targets.ts [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+      },
+    },
+    migrate: {
+      executor: '@nx-tools/nx-prisma:migrate',
+      options: {
+        schema: `${options.projectRoot}/prisma/schema.prisma`,
+      },
+    },
+    pull: {
+      executor: '@nx-tools/nx-prisma:rollback',
```
Comment (floross) [inline libs/schematics/src/generators/prisma-library/helpers/project-targets.ts]: pull

https://github.com/gperdomor/nx-tools/tree/main/packages/nx-prisma/src/executors

File: libs/schematics/src/generators/prisma-library/helpers/project-targets.ts [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+      },
+    },
+    push: {
+      executor: '@nx-tools/nx-prisma:seed',
+      options: {
+        schema: `${options.projectRoot}/prisma/schema.prisma`,
+      },
+    },
+    reset: {
+      executor: '@nx-tools/nx-prisma:seed',
```
Comment (floross) [inline libs/schematics/src/generators/prisma-library/helpers/project-targets.ts]: reset

**Review by maxmousse — APPROVED** [review]

**Pull request merged**